sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample1_AnalyzeConversationWithOptions.md

```diff
@@ -0,0 +1,34 @@
+# Ask a question
```

Comment:
There's already a ["Sample1_*" set of files](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample1_AnalyzeConversation.cs), so this - and subsequence samples, should be offset +1 e.g. Sample2_, etc. You might still keep them in the same order you have them, though. They are designed to be progressively harder / more advanced.